### PR TITLE
[WIP] Add style guide for references in `contributing.rst`

### LINKED ISSUE #2202: Standard citation style across docs
**Is your feature request related to a problem? Please describe.**
There is a mix of citation styles currently used in the documentation, both between pages and sometimes on the same page.

quick example:
[pvlib.atmosphere.get_relative_airmass](https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.atmosphere.get_relative_airmass.html)
Authors: full name (first+last), and first initial then surname
External links: some with DOI, some for which a DOI exists but is not included (e.g. ref 9).
Misc.: References in the list aren't always linked to something in the text.

Comparing with another page, e.g. [pvlib.irradiance.get_ground_diffuse](https://pvlib-python.readthedocs.io/en/stable/reference/generated/pvlib.irradiance.get_ground_diffuse.html), the naming style is last name then first name initial.

There are also differences in other aspects of the style e.g. page/volume numbers, whether a DOI is included, etc.

----
**Describe the solution you'd like**
1. Agree on a style
2. Add a section to the user guide and/or contributing guide advising users / potential contributors of the adopted referencing style, and additional suggestions such as to include a DOI if one is available, link reference to something in the text, etc.
3. Future PRs would adopt the agreed style, and old docs could be updated gradually in a few ways:
 3.1- Good first issue PRs for new contributors, e.g. at PVPMC workshops or other tutorials
 3.2- Dedicated PRs for anyone (something to do on those long-haul flights, right...)
 3.3- As existing functions that already contain citations are updated for whatever other purpose, updates to the references could be included in the same PR
 3.4- As new functions are added to the same packages, references for other functions from the same `.py` file could be updated 

The points in 3. are **just ideas/suggestions** and I hope we can discuss and decide on the best approach(es) together.

As for 1., I am not an expert on citation styles but I know there are many out there, e.g. IEEE, MLA, APA, Harvard... 
We could pick on out of there, or perhaps adopt a hybrid style of our own design, e.g. we like IEEE but prefer to keep the author list to one name + et al. for brevity.

---
**Question/suggestion**: Is adopting bibtex in the docs technically possible? The formatting could be set centrally and users would only need to update a `.bib` file somewhere (but we should ensure references still appear on the pages where they are cited, so maybe multiple `.bib` files would be required)

---
**Describe alternatives you've considered**
We could just continue without a set style but I think having, or at at least encouraging, a consistent citation style would make it easier for users to understand the citations, locate the sources, and even just make pvlib appear more professional. Maybe there's even be a legal(?)/ethical argument there where we need to ensure citations are up to a certain standard when we are sharing other people's work through pvlib. (not certain about that last one, just a thought)

---
**Additional context**
Perhaps we could start off with point 1 in the solutions: discuss whether this is a good idea and, if so, discuss what would be a good style to adopt.
If we get that far, I'd certainly be happy to resolve 2., and contribute to 3. :)

### PR DESCRIPTION
**Pull request by RDaxini**
Draft, not for review.

 - [X] Closes #2202
 - [ ] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [ ] Tests added
 - [ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.
 - [ ] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [ ] Pull request is nearly complete and ready for detailed review.
 - [ ] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by RDaxini** [comment]
Questions:
More example reference types? So far there's the general format for a journal paper, and two examples: journal paper and conference paper
Is it okay to link to a pdf? the pdf was found [here](https://ieeeaccess.ieee.org/guide-for-authors/preparing-your-article/)

**Comment by cwhanse** [comment]
> Questions: Too long? 

No, but a bit wordy in places. I can edit if you'd like.

> More (variety of) example reference types?

Yes. I think add a book or book chapter, and a technical report from some lab.

> Is it okay to link to a pdf? 

I don't see why not.

**Comment by RDaxini** [comment]
> > Questions: Too long?
> 
> No, but a bit wordy in places. I can edit if you'd like.

Sure, please go ahead

> > More (variety of) example reference types?
> 
> Yes. I think add a book or book chapter, and a technical report from some lab.

Okay will do

File: docs/sphinx/source/contributing/style_guide.rst [hunk 1]
```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
 status box at the bottom of the pull request.
 
 
+.. _references:
+
+References
+----------
+All pertinent information within a docstring should include a proper reference.
+In pvlib-python, we are transitioning to a standardised referencing system. We
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:78]:
```suggestion
In pvlib-python, we 
```

This page's scope is the reference style. The policy for what references are appropriate and where they are needed should be part of the Contributing guide, I think.

File: docs/sphinx/source/contributing/style_guide.rst [hunk 2]
```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+encourage using the `IEEE style <https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE_Reference_Guide.pdf>`_
+with numeric in-text citations, but overall the most important feature of all
+references is that they include sufficient information to make locating the
+original source as easy as possible. As a bare minimum, we advise including:
+
+* Author list (can be abbreviated with et al.)
+* Publication title
+* Publication source (journal title, laboratory name, etc.)
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:86]:
```suggestion
* Publisher (journal title, laboratory name, etc.)
```

File: docs/sphinx/source/contributing/style_guide.rst [hunk 3]
```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+.. _references:
+
+References
+----------
+All pertinent information within a docstring should include a proper reference.
+In pvlib-python, we are transitioning to a standardised referencing system. We
+encourage using the `IEEE style <https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE_Reference_Guide.pdf>`_
+with numeric in-text citations, but overall the most important feature of all
+references is that they include sufficient information to make locating the
+original source as easy as possible. As a bare minimum, we advise including:
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:82]:
```suggestion
with numeric in-text citations. At minimum, a reference should include:
```

File: docs/sphinx/source/contributing/style_guide.rst [hunk 4]
```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+* Publication title
+* Publication source (journal title, laboratory name, etc.)
+* Year of publication
+* DOI (if available)
+
+The recommended citation style for several media types is as follows:
+
+**Journal article**:
+
+    Author initials. Author Surname, "Title of article," abbreviated journal
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:94]:
```suggestion
    Author initials and surname, "Title of article," abbreviated journal
```

File: docs/sphinx/source/contributing/style_guide.rst [hunk 5]
```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+**Book**:
+
+    Author initials. Author Surname, “Chapter title” in Book Title, xth ed.
+    City of Publisher, (only U.S. State), Country: Abbrev. of Publisher, year,
+    ch. x, sec. x, pp. xxx–xxx.
+
+**Technical report**:
+
+    Author initials. Author Surname, “Report title” Abbrev. Name of Co.,
+    City of Co., Abbrev. State, Country, Rep. xxx, year
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:106]: "Co." can be confused with County. I think we should spell out "Company" or (better) use "publisher" again here. The NREL example does not list the city (Golden, CO).

File: docs/sphinx/source/contributing/style_guide.rst [hunk 6]
```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+           and Sons, 2006, ch. 1, sec. 1.5, pp.9-11
+
+    .. [4] R. Bird, and C. Riordan, “Simple solar spectral model for direct and
+           diffuse irradiance on horizontal and tilted planes at the earth’s
+           surface for cloudless atmospheres”, NREL, CO, USA, Technical Report
+           TR-215-2436, 1984, :doi:`10.2172/5986936`
+
+Things to note:
+
+* In text numeric citations require a number inside square brackets, followed
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:139]:
```suggestion
* In-text numeric citations are a number inside square brackets, followed
```

File: docs/sphinx/source/contributing/style_guide.rst [hunk 7]
```diff
@@ -70,6 +70,82 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+  by an underscore, e.g. ``[1]_``.
+* To include a DOI, you can use the existing ``:doi:``
+  `Sphinx role <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`_,
+  followed by the DOI string inside a set of backticks.
+* The citation formatting must be consistent within the same docstring, for
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:144]:
```suggestion
* The citation formatting should be consistent within the same docstring, for
```

File: docs/sphinx/source/contributing/style_guide.rst (hunk 1 above)
Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst:78]: Good point

Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst:78]: Still not worth mentioning the transitioning? I don't have a strong preference to include it, but I just thought it would explain why existing code that contributors see might not follow the recommended style. Still happy to delete that if you think so though 👍🏾

Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst:78]: Is it still not worth mentioning the transitioning bit?
I don't have a strong preference to include it, but I just thought it would explain why existing code that contributors see might not follow the recommended style. Still happy to delete that if you prefer though

Comment (cwhanse) [reply docs/sphinx/source/contributing/style_guide.rst:78]: That's also a good point. "pvlib-python is transitioning to a standard reference format." Does this work?

File: docs/sphinx/source/contributing/style_guide.rst [hunk 8]
```diff
@@ -74,9 +74,8 @@ status box at the bottom of the pull request.
 
 References
 ----------
-All pertinent information within a docstring should include a proper reference.
-In pvlib-python, we are transitioning to a standardised referencing system. We
-encourage using the `IEEE style <https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE_Reference_Guide.pdf>`_
+pvlib-python is transitioning to a standard reference format. We encourage
```
Comment (cwhanse) [inline docs/sphinx/source/contributing/style_guide.rst:77]: Re-reading this, we say "standard" reference format but then "encourage" use of IEEE, leaving me unsure if the IEEE style is the standard for pvlib.

If so, then "pvlib-python is transitioning to the IEEE reference style" and "New code should use the IEEE style" would be more clear.

If we are accepting other formats, then "encourage" is OK but replace "standard" with "more consistently formatted references" I'm not in favor of this idea and I thought we agreed that IEEE is preferred.

Comment (AdamRJensen) [reply docs/sphinx/source/contributing/style_guide.rst:77]: There should only be one correct way. Let's require IEEE for new contributions

Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst:77]: All good points

> I'm not in favor of this idea and I thought we agreed that IEEE is preferred.

Me neither, and yes we did

I was thinking about [this comment](https://github.com/pvlib/pvlib-python/issues/2202#issuecomment-2346879058) (the recommend not insist bit) when I used the words recommend and encourage.

I agree the current wording is unclear and should be improved though.

> "pvlib-python is transitioning to the IEEE reference style" and "New code should use the IEEE style"

I am okay with the above.

I might slightly prefer: "pvlib-python is transitioning to a consistent reference style. New code should use the IEEE style."
Possibility: "new contributions" instead of "new code"? Replace one or both of the "style" with "format"?

Comment (cwhanse) [reply docs/sphinx/source/contributing/style_guide.rst:77]: > "pvlib-python is transitioning to the IEEE reference style, which should be used in all new contributions"

+1

> "new contributions" instead of "new code"

Contributions is better

**Comment by AdamRJensen** [comment]
I think we should encourage users not to format it manually, but perhaps use something like this: https://citation.crosscite.org/

Also, the DOI should be required if there is one.

Last, might be a good idea to link to the official IEEE description: https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE_Reference_Guide.pdf

**Comment by RDaxini** [comment]
> I think we should encourage users not to format it manually, but perhaps use something like this: https://citation.crosscite.org/
>
Do we want pvlib to endorse one particular citation generator though...? I think most people would be aware of such sites and just go ahead and use one themselves if they want to.

> Also, the DOI should be required if there is one.

Already included, see line 85

> Last, might be a good idea to link to the official IEEE description: https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE_Reference_Guide.pdf

Already included, see line 78 in the original, 77 since the last commit

> The official guide (above) states the following, which we should abide by:
> 
> > IEEE publications must list names of all authors, up to six names. If there are more than six names listed, use the primary author’s name followed by “et al.”

Have now added it in (line 81)

File: docs/sphinx/source/contributing/style_guide.rst [hunk 9]
```diff
@@ -70,6 +70,76 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+    are also some conference papers linked to pvlib, for example [2]_.
+    
+    Other types of reference you may find in the pvlib-python documentation
+    include books [3]_ and technical reports [4]_.
+
+    References
+    ----------
+    .. [1] K. Anderson, C. Hansen, W. Holmgren, A. Jensen, M. Mikofski,
+           and A Driesse. “pvlib python: 2023 project update.” J. Open Source
+           Softw. 8(92), 5994, (2023). :doi:`10.21105/joss.05994`
```
Comment (kandersolar) [inline docs/sphinx/source/contributing/style_guide.rst:118]: This doesn't seem to match IEEE style

Comment (cwhanse) [reply docs/sphinx/source/contributing/style_guide.rst:118]: This part? " 8(92), 5994, (2023)"

Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst]: Please help me check the last commit before I mark this as resolved. I swapped out the brackets for vol./no. and commas.

File: docs/sphinx/source/contributing/style_guide.rst [hunk 10]
```diff
@@ -70,6 +70,77 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+    .. [2] J. S. Stein, “The Photovoltaic Performance Modeling Collaborative
+           (PVPMC),” In Proc. 38th IEEE Photovoltaic Specialists Conference
+           (PVSC), Austin, TX, USA, 2012, pp. 3048-3052,
+           :doi:`10.1109/PVSC.2012.6318225`
+
+    .. [3] J. A. Duffie and W. A. Beckman, “Solar Radiation" in Solar
+           Engineering of Thermal Processes, 3rd ed, New York, USA, J. Wiley
+           and Sons, 2006, ch. 1, sec. 1.5, pp.9-11
+
+    .. [4] R. Bird, and C. Riordan, “Simple solar spectral model for direct and
```
Comment (kandersolar) [inline docs/sphinx/source/contributing/style_guide.rst:130]:
```suggestion
    .. [4] R. Bird and C. Riordan, “Simple solar spectral model for direct and
```

File: docs/sphinx/source/contributing/style_guide.rst [hunk 11]
```diff
@@ -70,6 +70,77 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+    This is the recommended citation for the pvlib-python project [1]_. There
+    are also some conference papers linked to pvlib, for example [2]_.
+    
+    Other types of reference you may find in the pvlib-python documentation
+    include books [3]_ and technical reports [4]_.
+
+    References
+    ----------
+    .. [1] K. Anderson, C. Hansen, W. Holmgren, A. Jensen, M. Mikofski,
+           and A Driesse. “pvlib python: 2023 project update.” J. Open Source
```
Comment (kandersolar) [inline docs/sphinx/source/contributing/style_guide.rst:117]: Let's replace the "fancy" quotation mark characters with normal ones `"` in each of these four examples

**Review by kandersolar — APPROVED** [review]
This is OK with me

File: docs/sphinx/source/contributing/style_guide.rst (hunk 10 above)
Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst:130]: From what I have seen in the style guide, a comma is not used when there are only two authors
Attached example
image

Comment (kandersolar) [reply docs/sphinx/source/contributing/style_guide.rst]: I think we are in agreement; I am suggesting to remove such a comma

Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst]: Yes sorry my mistake! I noticed just after so deleted my comment and applied the change. I tried adding you as a co-author to the commit on GH desktop because the conversation became outdated but forgot to hit search user before commit 🤦🏾‍♂️

File: docs/sphinx/source/contributing/style_guide.rst [hunk 12]
```diff
@@ -70,6 +70,77 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+           surface for cloudless atmospheres", NREL, Golden, CO, USA, Technical
+           Report TR-215-2436, 1984, :doi:`10.2172/5986936`
+
+Things to note:
+
+* In-text numeric citations are a number inside square brackets, followed
+  by an underscore, e.g. ``[1]_``.
+* To include a DOI, you can use the existing ``:doi:``
+  `Sphinx role <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`_,
+  followed by the DOI string inside a set of backticks.
```
Comment (echedey-ls) [inline docs/sphinx/source/contributing/style_guide.rst:141]: I wonder whether this link is confusing since `:doi:` role does not appear in that page.

```suggestion
* To include a DOI link, use the custom ``:doi:`` role, e.g.,
  |:doi:`10.21105/joss.05994`| will render as `DOI: 10.21105/joss.05994 <dx.doi.org/10.21105/joss.05994>`_.
```

**Review by echedey-ls — APPROVED** [review]
I feel like so many details aren't necessary. In fact, so many details on how IEEE work may encourage contributors to manually type the citations. But I can live with that.

> Do we want pvlib to endorse one particular citation generator though...? 

I'd say we don't want to, but citing "Use of a citation generator is recommended" at least somewhere would be nice. Whether or not to provide a link... dunno. I wouldn't put it just as there are plenty of them.

In addition, maybe add Zotero, other bib managers or the published paper webpage as sources to get the IEEE citation from.


LGTM nevertheless, this section would definitely be remembered for whoever reads it!

**Review by cwhanse — APPROVED** [review]

**Comment by AdamRJensen** [comment]
On a couple of recent PRs to another repo I've been using the IEEE citation format... and I really dislike how the first name initial comes first.  It seems that most pvlib functions already are using a somewhat fairly consistent format - anyone know what that is?

**Comment by cwhanse** [comment]
> It seems that most pvlib functions already are using a somewhat fairly consistent format - anyone know what that is?

Likely, just imitating other references. There hasn't been a citation style specified or requested before.

File: docs/sphinx/source/contributing/style_guide.rst (hunk 12 above)
Comment (RDaxini) [reply docs/sphinx/source/contributing/style_guide.rst:141]: > I wonder whether this link is confusing since `:doi:` role does not appear in that page.

I think it's okay if `:doi:` is not there as only phrase "sphinx role" is linked; the purpose was just to provide some context/info in case the reader is unfamiliar with the term "role" --- I accept it is not essential, but I was just trying to be helpful 😅

I am not opposed to revising further, but I am unsure about the suggested revision... Save for the "render as..." I think this is covered by the example?

**Comment by RDaxini** [comment]
@AdamRJensen I also think the initial/name order is a slightly odd, it's not what I am used to anyway, but it's not a dealbreaker for me. I think IEEE is still a good choice. Is there another style you'd recommend?

File: docs/sphinx/source/contributing/style_guide.rst (hunk 12 above)
Comment (echedey-ls) [reply docs/sphinx/source/contributing/style_guide.rst:141]: At least add the word `custom` in front of it, please.

```suggestion
* To include a DOI, you can use pvlib's custom ``:doi:``
  `Sphinx role <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`_,
  followed by the DOI inside a set of backticks.
```

**Comment by mikofski** [comment]
> I feel like so many details aren't necessary. In fact, so many details on how IEEE work may encourage contributors to manually type the citations. But I can live with that.
> 
> > Do we want pvlib to endorse one particular citation generator though...?
> 
> I'd say we don't want to, but citing "Use of a citation generator is recommended" at least somewhere would be nice. Whether or not to provide a link... dunno. I wouldn't put it just as there are plenty of them.
> 
> In addition, maybe add Zotero, other bib managers or the published paper webpage as sources to get the IEEE citation from.
> 
> LGTM nevertheless, this section would definitely be remembered for whoever reads it!

What would be cool is a Zotero group or library that we could share with pvlib users

File: docs/sphinx/source/whatsnew/v0.11.2.rst [hunk 13]
```diff
@@ -15,6 +15,10 @@ Enhancements
 Documentation
 ~~~~~~~~~~~~~
 * Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
+  :py:func:`~pvlib.pvsystem.dc_ohmic_loss` for clarity. (:issue:`1601`, :pull:`2229`)
+* Added a new reference style guide to the contributing page.
```
Comment (echedey-ls) [inline docs/sphinx/source/whatsnew/v0.11.2.rst:19]:
```suggestion
* Added a new bibliographic reference style guide to the contributing page.
```
Plain `reference style guide`, reads like a style guide of reference for code style? IDK, it's good as it is.
May as well link to it with ```:ref:`references` ```.

**Review by echedey-ls — COMMENTED** [review]
> What would be cool is a Zotero group or library that we could share with pvlib users

@mikofski I'm completely ignorant on that feature of Zotero, but the bibtex file with references in pvlib would be super for that? I mean, centralized in the repo so nobody ever forgets about updating it, in the same PRs that get accepted, and for those non-Zotero users a webpage with all the references to find if a paper is or not relevant in pvlib / a model is a good candidate.

File: docs/sphinx/source/whatsnew/v0.11.2.rst (hunk 13 above)
Comment (AdamRJensen) [reply docs/sphinx/source/whatsnew/v0.11.2.rst:19]:
```suggestion
* Added a new reference citation style guide to the contributing page.
```
I think "citation" is more clear than "bibliographic"

**Comment by AdamRJensen** [comment]
> @AdamRJensen I also think the initial/name order is a slightly odd, it's not what I am used to anyway, but it's not a dealbreaker for me. I think IEEE is still a good choice. Is there another style you'd recommend?

I would vote for the APA style. Here's an example:

**IEEE Style:**
C. Gueymard, "Analysis of monthly average atmospheric precipitable water and turbidity in Canada and northern United States," Solar Energy, vol. 53, no. 1, pp. 57-71, 1994.

**APA Style:**
Gueymard, C. (1994). Analysis of monthly average atmospheric precipitable water and turbidity in Canada and northern United States. Solar Energy, 53(1), 57-71. DOI

Vote this post up if you agree with APA and vote down if you prefer IEEE.

File: docs/sphinx/source/contributing/style_guide.rst [hunk 14]
```diff
@@ -70,6 +70,77 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
...
+    City of publisher, Abbrev. State, Country, Rep. xxx, year
+
+The example below shows how to cite material and generate a reference list
+using the IEEE style in a docstring::
+
+    This is the recommended citation for the pvlib-python project [1]_. There
+    are also some conference papers linked to pvlib, for example [2]_.
+    
+    Other types of reference you may find in the pvlib-python documentation
+    include books [3]_ and technical reports [4]_.
```
Comment (AdamRJensen) [inline docs/sphinx/source/contributing/style_guide.rst:112]: I suggest adding a website also.

**Comment by cwhanse** [comment]
> Vote this post up if you agree with APA and vote down if you prefer IEEE.

Personally, I think the APA style is easier to read. But I'm voting for IEEE because 1) IEEE publishes the many of the references cited, 2) IEEE lets one export the citation but only in the IEEE format. If we choose APA, then I need to edit the citation rather than just copy/paste.

File: docs/sphinx/source/whatsnew/v0.11.2.rst (hunk 13 above)
Comment (RDaxini) [reply docs/sphinx/source/whatsnew/v0.11.2.rst:19]: > I think "citation" is more clear than "bibliographic"

Agreed

But then "reference citation style guide" as per the [suggestion](https://github.com/pvlib/pvlib-python/pull/2226#discussion_r1797250422), or just "citation style guide"? I lean towards the latter

Comment (AdamRJensen) [reply docs/sphinx/source/whatsnew/v0.11.2.rst:19]: Latter is simpler, I agree

**Review by AdamRJensen — APPROVED** [review]
This is a great addition, very helpful!

@RDaxini thumb up this post and I'll merge this PR in a couple of days.

File: docs/sphinx/source/whatsnew/v0.11.2.rst [hunk 15]
```diff
@@ -15,7 +15,9 @@ Enhancements
 Documentation
 ~~~~~~~~~~~~~
 * Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
-  :py:func:`~pvlib.pvsystem.dc_ohmic_losses` for clarity. (:issue:`1601`, :pull:`2229`)
+  :py:func:`~pvlib.pvsystem.dc_ohmic_loss` for clarity. (:issue:`1601`, :pull:`2229`)
```
Comment (kandersolar) [inline docs/sphinx/source/whatsnew/v0.11.2.rst:18]: This change should be reverted, I think.  `..._losses` is correct.

Comment (RDaxini) [reply docs/sphinx/source/whatsnew/v0.11.2.rst:18]: Ah I was not trying to change the existing entry... this incomplete entry ":py:func:`~pvlib.pvsystem.dc_ohmic_losses` for clarity. (:issue:`1601`, :pull:`2229`)" just appeared, not attached to an entry, after I updated the file....
So this line should be replacing the line in the entry with "loss"?
